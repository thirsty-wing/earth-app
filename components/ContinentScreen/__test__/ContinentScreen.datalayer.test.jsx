import ContinentScreenDatalayer from "../ContinentScreen.datalayer";
import { shallow } from "../../../utils/test/shallow";


jest.mock("apollo-fetch", () => ({
  createApolloFetch: () => jest.fn(),
}));

it("should render", () => {

  const route = {
    params: {
      code: "GW",
    },
  };

  const content = shallow(
    <ContinentScreenDatalayer
      route={route}
      navigation={{}}
    />
  );

  expect(content).toMatchSnapshot();
});
