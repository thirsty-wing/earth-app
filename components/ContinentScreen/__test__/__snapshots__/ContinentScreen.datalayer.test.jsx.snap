// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<ContinentScreen
  data={{}}
  navigation={{}}
  responseStatus="LOADING"
/>
`;
